### PR TITLE
Fix negative next check value when both active and passive checks are disabled

### LINKED ISSUE #403: CheckStatistics: Reflect disabled active checks
## Describe the bug
Disabling active checks afftects the “next check” value from the check statistics widget. Currently this leads to negative values for the “next check” or when the view is updated wrong values

## To Reproduce
1. Open a service/host detail view
2. disabled active checks
3. Wait until the check interval is over to see the negative “Next Check” value

## Expected behavior
We should consider the Check Statistics widget to reflect that active checks are disabled.

## Screenshots
Screen Shot 2021-10-20 at 13 55 45

### PR DESCRIPTION
**Pull request by raviks789**
resolves #403

### CODE REVIEW

File: library/Icingadb/Widget/Detail/CheckStatistics.php [hunk 1]
```diff
@@ -104,13 +104,6 @@ protected function assembleBody(BaseHtmlElement $body)
             'style' => sprintf('left: %F%%', $hPadding + $leftNow),
         ]);
 
-        $timeline->add([
-            $markerLast,
-            $markerNow,
-            $markerNext,
-            $overdueBar
-        ]);
```
Comment (nilmerg) [inline library/Icingadb/Widget/Detail/CheckStatistics.php:112]: Please restore this, you don't have to reorder it.

File: library/Icingadb/Widget/Detail/CheckStatistics.php [hunk 2]
```diff
@@ -132,18 +125,52 @@ protected function assembleBody(BaseHtmlElement $body)
                     : (new EmptyState(t('n. a.')))->setTag('span')
             )
         );
+
+        if ($this->isChecksDisabled()) {
+            $markerLast->addAttributes([
+                'class' => 'state-' . $this->object->state->getStateText()
+            ]);
```
Comment (nilmerg) [inline library/Icingadb/Widget/Detail/CheckStatistics.php:132]: I don't think I talked about this change with @flourish86, that's just an artifact in the mockup. I see absolutely no reason to introduce this change here, it has nothing to do with active/passive checks being off.

File: public/css/mixin/progress-bar.less [hunk 3]
```diff
@@ -177,3 +177,49 @@
...
 }
+
+.progress-bar.checks-disabled {
+  .above {
+    .checks-disabled-text {
+      display: block;
+      width: 100%;
+      text-align: center;
+    }
+  }
```
Comment (nilmerg) [inline public/css/mixin/progress-bar.less:188]: This is no general style. Please move this to `check-statistics.less`.

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: public/css/mixin/progress-bar.less (hunk 3 above)
Comment (raviks789) [reply public/css/mixin/progress-bar.less]: The disabled note is moved to overlay container now.

**Review by nilmerg — APPROVED** [review]

**Pull request merged**